spring-cloud-stream-binders/spring-cloud-stream-binder-rabbit/src/main/java/org/springframework/cloud/stream/binder/rabbit/RabbitConsumerProperties.java

```diff
@@ -74,6 +76,7 @@ public void setAcknowledgeMode(AcknowledgeMode acknowledgeMode) {
 		this.acknowledgeMode = acknowledgeMode;
 	}
 
+	@Min(value = 1, message = "Max Concurrency should be greater than or equal to 1")
```

Comment:
might be worth defining a constant for: "should be greater than or equal to 1"

in fact, I'm wondering if it would be better to say "must be a positive value" (and for the >=0 case, "must not be negative)